src/Microsoft.DocAsCode.Build.Engine/MarkdownProviders/JsonServiceProviders.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.Engine.MarkdownProviders
```

Comment:
Yes ,agree

---
In reply to: [74545324](https://github.com/dotnet/docfx/pull/602#discussion_r74545324) [](ancestors = 74545324)